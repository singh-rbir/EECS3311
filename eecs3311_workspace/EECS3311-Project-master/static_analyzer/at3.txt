add_generalized_and
end_set_enumeration 
add_boolean_constant(true)
type_check
evaluate 

reset

start_set_enumeration 
end_set_enumeration
type_check
evaluate 