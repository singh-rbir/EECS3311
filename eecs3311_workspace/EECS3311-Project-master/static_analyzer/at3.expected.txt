  Expression currently specified: ?
  Report: Expression is initialized.
->add_generalized_and
  Expression currently specified: (&& ?)
  Report: OK.
->end_set_enumeration
  Expression currently specified: (&& ?)
  Report: Error (Set enumeration is not being specified).
->add_boolean_constant(True)
  Expression currently specified: (&& True)
  Report: OK.
->type_check
  Expression currently specified: (&& True)
  Report: (&& True) is not type-correct.
->evaluate
  Expression currently specified: (&& True)
  Report: Error (Expression is not type-correct).
->reset
  Expression currently specified: ?
  Report: OK.
->start_set_enumeration
  Expression currently specified: {?}
  Report: OK.
->end_set_enumeration
  Expression currently specified: {?}
  Report: Error: (Set enumeration must be non-empty).
->type_check
  Expression currently specified: {?}
  Report: Error (Expression is not yet fully specified).
->evaluate
  Expression currently specified: {?}
  Report: Error (Expression is not yet fully specified).
